mobile/Dockerfile

```diff
@@ -29,18 +29,11 @@ ENV PATH ${PATH}:/opt/android/tools/bin
 RUN sdkmanager --update \
     && yes | sdkmanager \
     "build-tools;26.0.1" \
-    "platforms;android-25"
+    "platforms;android-26"
 
 # Install Cordova, and opt out of telemetry reporting. If we don't opt-out now,
 # then the user will face an interactive prompt when building the image later.
 RUN npm install -g cordova \
     && cordova telemetry off
 
 WORKDIR /mobile
-# This can be overridden by the user, but assumes that the repo's mobile/
```

Comment:
Thanks - I've been meaning to remove this. It was handy for me during development, but I didn't really mean to merge it!